Dockerfile

```diff
@@ -8,14 +10,17 @@ COPY package.json package-lock.json /usr/src/app/
 # Without the badge-maker package.json and CLI script in place, `npm ci` will fail.
 COPY badge-maker /usr/src/app/badge-maker/
 
-RUN npm install -g "npm@^10"
+COPY . /usr/src/app
+
 # We need dev deps to build the front end. We don't need Cypress, though.
-RUN NODE_ENV=development CYPRESS_INSTALL_BINARY=0 npm ci
+RUN NODE_ENV=development CYPRESS_INSTALL_BINARY=0 npm ci \
+    && npm run build \
+    && npm prune --omit=dev \
+    && npm cache clean --force \
+    && rm -rf node_modules \
```

Comment:
i tested and this line does make the difference. 

after some testing i found out that the cache is 815mb inside the container even after `npm cache clean --force` is run.
`npm cache clean --force` cleans download cache from ~/.npm/_cache....
but our size increase comes from `node_module/.cache`
the tool that puts data in this `node_module/.cache` is webpack.

we can set a webpack config to avoid cache, but this will be slower for development ([see also docs](https://webpack.js.org/configuration/cache/))
but more reasonable imo is just removing the cache here.

Reply:
What about a webpack config that is just used when building the docker container?

(I got 0 clue about webpack)